app/views/content_items/html_publication.html.erb

```diff
@@ -64,7 +64,7 @@
       </div>
     </div>
 
-    <div class="main-content-container<% unless @content_item.contents.any? %> offset-empty-contents-list<% end %>">
+    <div class="govuk-grid-column-three-quarters-from-desktop contents-container<% unless @content_item.contents.any? %> offset-empty-contents-list<% end %>">
```

Comment:
Why is this using the `govuk-grid-column-three-quarters-from-desktop` class over the `govuk-grid-column-three-quarters` class?

Reply:
I kept it consistent with the contents-list column which uses the 1/4 from desktop class. I assume that the original design for this layout wanted to ensure a full width display on tablets and only switch to columns on desktop?

Reply:
FYI, these are the differences between those two classes:

**Three Quarters**

```
.govuk-grid-column-three-quarters{
	box-sizing:border-box;
	width:100%;
	padding:0 15px
}
@media (min-width:40.0625em){
	.govuk-grid-column-three-quarters{
		width:75%;
		float:left}
	}
}
```

**Three Quarters from Desktop**

```
.govuk-grid-column-three-quarters-from-desktop{
	box-sizing:border-box;
	padding:0 15px
}
@media (min-width:48.0625em){
	.govuk-grid-column-three-quarters-from-desktop{
		width:75%;
		float:left
	}
}
```

Reply:
Looks like. Having tested it, it seems the difference is that with the `-from-desktop` classes it collapses to a single column at 768px, but without them it collapses at 640px, which looks a little squashed, so I think stick with the classes as is 👍